doc/dev/developer_guide/index.rst

```diff
@@ -915,9 +915,9 @@ the following recommendations should be followed:
 
 * **50 <= Priority < 75:** Use this if you are testing a particular feature/fix and running fewer than about 25 jobs.
 
-* **75 <= Priority < 100:** Tech Leads will regularly schedule integration tests with this priority to verify pull requests against master.
+* **75 <= Priority < 100:** Tech Leads will regularly schedule integration tests with this priority to verify pull requests against master and point releases.
```

Comment:
As discussed in the private email thread, -1 on this change.

Reply:
We can note that for `50 <= Priority < 75` that **urgent** release testing may also be done.

Reply:
corrected per our discussion in the private email thread